### PR TITLE
boa: fix the install script python

### PR DESCRIPTION
**Pull request by yorkie**
In this PR, we use the specific Python binary for interrupting the Pip and Conda script to avoid environment issues.

### CODE REVIEW

File: packages/boa/lib/index.js [hunk 1]
```diff
@@ -10,6 +10,15 @@ const DelegatorLoader = require('./delegator-loader');
 // internal symbols
 const IterIdxForSeqSymbol = Symbol('The iteration index for sequence');
 
+// read the conda path from the .CONDA_INSTALL_DIR
+// eslint-disable-next-line no-sync
+const condaPath = fs.readFileSync(path.join(__dirname, '../.CONDA_INSTALL_DIR'), 'utf8');
```
Comment (FeelyChau) [inline packages/boa/lib/index.js:15]:
```suggestion
const condaPath = fs.readFileSync(path.join(__dirname, '..', '.CONDA_INSTALL_DIR'), 'utf8');
```
for more cross-platform adaptability.

Comment (FeelyChau) [reply packages/boa/lib/index.js:15]: And could we use async function？

Comment (yorkie) [reply packages/boa/lib/index.js:15]: unfortunately, we have to use this function here, because the top-level await is not supported.

Comment (yorkie) [reply packages/boa/lib/index.js:15]: > adaptability

The `path.join` resolved the cross-platform issue in this way, you could try:

```
> path.win32.join('foobar', '123/test')
'foobar\\123\\test'
```

**Pull request merged**